tests/hypothesis/to_unit_test.py

```diff
@@ -14,32 +14,53 @@
 _max_examples = 1000
 
 
-@given(st.floats(min_value=1e-300, max_value=1e300))
 @settings(max_examples=_max_examples)
-def test_to_unit_value_1(x):
+def _to_unit_value_one(x):
     unit = sc.Unit(f'{x:.14E}')
     var = sc.scalar(x, unit='')
     result = sc.to_unit(var, unit)
     assert result.unit == unit
     # to_unit is rounding to 1e-12 so generally we cannot expect more precision
-    assert pytest.approx(result.value, abs=0.0, rel=1e-12) == 1.0
+    assert pytest.approx(result.value, abs=0.0, rel=1.1e-12) == 1.0
```

Comment:
A minor thing: applying `approx` to the RHS (the expected value) produces better error messages. Using it on the LHS mixes up 'observed' and 'expected' results.